### ISSUE TITLE
streamlit-ui.py missing?

### ISSUE THREAD

**Issue by aptonline** [robmarkcole/deepstack-ui#47]
Receiving the following error after upgrading to 0.8 from 0.6.

```
Error: Invalid value: File does not exist: streamlit-ui.py | stdout
Usage: streamlit run [OPTIONS] TARGET [ARGS]...

```

**Comment by robmarkcole** [comment]
renamed `deepstack-ui.py`

**Comment by aptonline** [comment]
sorry how has this fixed the issue for the currently deployed docker image? Do I need to wait for the next release?

**Comment by robmarkcole** [comment]
Make sure you Docker pull latest. If this is still an issue please list the steps to reproduce it

**Comment by aptonline** [comment]
I pulled latest docker image when 0.8 was released but that's when I got the error. I manually removed the image and re-downloaded and that seems to resolve the issue.  Now Face detection times out for some reason.  I have Deepstack running on a Synology with both `VISION-Face` and `VISION-Detection` set to `True` with `MODE` set to `High`.  I'll try the new debug option to see if I can see what's going on.
